bin/build-macos-universal.sh

```diff
@@ -114,9 +103,8 @@ if [ ! -e "${DIR_BUILD}/blazingmq/.complete" ]; then
         -DFLEX_ROOT="${FLEX_ROOT}"
         -G "Ninja")
     cmake -B "${DIR_BUILD}/blazingmq" -S "." "${CMAKE_OPTIONS[@]}"
-    cmake --build "${DIR_BUILD}/blazingmq" -j 16 --target bmq
-    cmake --install "${DIR_BUILD}/blazingmq" --component mwc-all
-    cmake --install "${DIR_BUILD}/blazingmq" --component bmq-all
+    cmake --build "${DIR_BUILD}/blazingmq" -j 16 --target all
+    cmake --install "${DIR_BUILD}/blazingmq"
```

Comment:
Will the default install target also install the mqb headers?

Reply:
Looks like it:

~~~
XNCM933LQR:blazingmq-sdk-python pniedzielski$ ls ${DIR_INSTALL}/include/ | grep mqb
mqbcfg_brokerconfig.h
mqbcfg_messages.h
mqbplug_pluginfactory.h
mqbplug_plugininfo.h
mqbplug_pluginlibrary.h
mqbplug_pluginmanager.h
mqbplug_plugintype.h
mqbplug_statconsumer.h
mqbscm_version.h
mqbscm_versiontag.h
mqbstat_brokerstats.h
mqbstat_clusterstats.h
mqbstat_domainstats.h
mqbstat_printer.h
mqbstat_queuestats.h
mqbstat_statcontroller.h
~~~